css-two-column-layout/styles.css

```diff
@@ -0,0 +1,83 @@
+body {
+  font-family: Karla, sans-serif;
+}
+
+.container {
+  max-width: 800px;
+  margin: 0 auto;
+}
+
+.row {
+  display: flex;
+  flex-wrap: wrap;
+  border-spacing: 0;
+  border-collapse: collapse;
+}
+
+.main-header {
+  letter-spacing: 3.3px;
+  margin: 45px 0 15px 0;
+  font-size: 2.5rem;
+  font-weight: 570;
+}
+
+.header-paragraph {
+  font-size: 12px;
+  color: gray;
+  padding: 0 7rem;
+  line-height: 18px;
+}
+
+.column-one-half {
+  position: relative;
+  left: 30px;
+  flex-basis: calc(100% / 2);
+  padding: 25px 50px 0;
+}
+
+.image {
+  width: 95px;
+  position: relative;
+  right: 6rem;
```

Comment:
Hmm, why is this large movement necessary? What happens when this isn't applied? Feel free to message me if you aren't sure how to solve this effectively.